server.ts

```diff
@@ -64,6 +64,7 @@ export function app(): express.Express {
           'recaptcha.net',
           'www.recaptcha.net',
           'www.youtube.com',
+          'www.youtube-nocookie.com.com',
```

Comment:
This might be a good opportunity to split out `frame-src`, with the caveat that we would need to comb through our other "defaults" and work out which are in fact used for frames, fonts, something else or redundant. A little fiddly but we'd end up with a slightly clearer security situation if we do it.

Reply:
I've moved youtube into frames-src in 6e8e3bf9 . Although confusingly the page seems seems to work fine and load the iframe on my local even if I comment out all youtube mentions entirely in server.ts